packages/circularprogress/src/index.d.ts

```diff
@@ -1,24 +1,39 @@
-import { ForwardRefExoticComponent, RefAttributes } from 'react'
+import { ForwardRefExoticComponent, HTMLAttributes, RefAttributes } from 'react'
 
-interface Styling {
-  css?: Record<string, string | number>
-  style?: Record<string, string | number>
-  className?: string
-}
+declare const CircularProgress: RefForwardingComponent<
+  Props,
+  HTMLDivElement,
+  Statics
+>
+
+export default CircularProgress
 
-export interface CircularProgressProps {
-  size?: 'small' | 'medium'
+export interface Props {
```

Comment:
The reason I named this `CircularProgressProps` is to follow a convention of `{ComponentName}Props`, which would provide users with a consistent way to augment the props interface if they need to, like so:

```typescript
declare module '@pluralsight/ps-design-system-tab' {
    interface ListItemProps {
        name: string                          //add "name" prop to ListItem
    }
}
```

It feels redundant with single-export components, but makes more sense when you consider things like `Tab` that have multiples, and this way there would only be one convention to remember.

That said, now that you've added the `HTMLAttributes` in to the props, does that cover all the props that a user might provide (and would actually get forwarded on?) If so, then there's probably less need to augment, and I don't really care either way.

Reply:
I like the `{ComponentName}Props` convention.